ang/civicase.ang.php

```diff
@@ -235,6 +235,14 @@
 $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
 $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
 
+// Retrieve civicase webform URL
+$allowCaseWebform = Civi::settings()->get('civicaseAllowCaseWebform');
+$webformURl = NULL;
+if ($allowCaseWebform) {
+  $webformURl = Civi::settings()->get('civicaseWebformUrl');
+}
```

Comment:
Do you think it's possible to do this in PHP?

```php
$options['webformUrl'] = $allowCaseWebform
  ? Civi::settings()->get('civicaseWebformUrl')
  : NULL;
```

Reply:
Yes we can!